sdk/search/search/CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## 11.0.0-preview.2 (Unreleased)
 
+- Support for index management operations via the `SearchServiceClient`.
```

Comment:
Nit: "via" -> "using". More conversational, which is the guidelines for general user documentation.